src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -89,6 +89,12 @@ void UpdateModeBaseCurrentModeToOnMode(EndpointId endpoint)
 
 #endif // MATTER_DM_PLUGIN_MODE_BASE
 
+template <typename EnumType>
+bool isKnownEnumValue(EnumType value)
```

Comment:
```suggestion
bool IsKnownEnumValue(EnumType value)
```
is how things generally get named in this code at this point, I think....